src/pykiso/test_setup/config_registry.py

```diff
@@ -39,11 +39,12 @@ def register_aux_con(cls, config: dict) -> None:
         """
         ConfigRegistry._linker = DynamicImportLinker()
         ConfigRegistry._linker.install()
-        for connector, con_details in config["connectors"].items():
-            cfg = con_details.get("config") or dict()
-            ConfigRegistry._linker.provide_connector(
-                connector, con_details["type"], **cfg
-            )
+        if not config["connectors"] is None:
```

Comment:
If you assign {} to the connectors entry in config_parser if none is defined you can get rid of this additional check

Reply:
Done